flink-doris-connector/pom.xml

```diff
@@ -379,10 +391,10 @@ under the License.
                     <version>3.2.4</version>
                     <configuration>
                         <relocations>
-                            <relocation>
-                                <pattern>org.apache.arrow</pattern>
-                                <shadedPattern>org.apache.doris.shaded.org.apache.arrow</shadedPattern>
-                            </relocation>
+<!--                            <relocation>-->
+<!--                                <pattern>org.apache.arrow</pattern>-->
+<!--                                <shadedPattern>org.apache.doris.shaded.org.apache.arrow</shadedPattern>-->
+<!--                            </relocation>-->
```

Comment:
Is there any reason for removing the shade here?

Reply:
This configuration will cause the wrong package name when loading the FlightSqlDriver driver ![image](https://github.com/user-attachments/assets/b8d69ca6-2e95-4a01-a6d7-9224a39a8433)